sdk/azcore/internal/pipeline/request.go

```diff
@@ -157,8 +157,7 @@ func (req *Request) Close() error {
 
 // Clone returns a deep copy of the request with its context changed to ctx.
 func (req *Request) Clone(ctx context.Context) *Request {
-	r2 := Request{}
-	r2 = *req
+	r2 := *req
```

Comment:
Just checking - is this just a lint fix or is there something 1.18-interesting in here?

Reply:
It's linter clean-up found in the latest version of the linter.